components/overview/AmountInput.js

```diff
@@ -12,12 +11,30 @@ const AmountInputDefault = ({
 	availableBalance,
 	onChange,
 	networkInfo,
+	isExistingUser,
+	currentDate,
+	lastDiscountDate,
 }) => {
 	const maxAmount =
```

Comment:
@sahilnanda1995 can you check if the logic here is correct? i am not good with ternaries.

Reply:
yeah the logic seems correct, but the bigger issue is [this](https://github.com/yieldscan/yieldscan-frontend/pull/134/files#r720849650)

Reply:
addressed the issue